.github/workflows/auditbeat-macos-unit-tests.yml

```
@@ -0,0 +1,66 @@
+name: auditbeat-macos-unit-tests
+
+env:
+  BEAT: "auditbeat"
+
+on:
+  pull_request:
+    paths:
+      - auditbeat/
+      # OSS
+      - go.mod
+      - dev-tools/
+      - libbeat/**
+      # Workflow
+      - .github/workflows/auditbeat-macos-unit-tests.yml
+      - .github/actions/unit-test/action.yml
+  push:
+    branches:
+      - main
+    paths:
+      - auditbeat/
+      # OSS
+      - go.mod
+      - dev-tools/
+      - libbeat/**
+      # Workflow
+      - .github/workflows/auditbeat-macos-unit-tests.yml
+      - .github/actions/unit-test/action.yml
+
+permissions:
+  contents: read
+
+## Concurrency only allowed in the main branch.
+## So old builds running for old commits within the same Pull Request are cancelled
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
+
+jobs:
+  unit-tests-branch:
+    name: Unit tests ${{ matrix.os }}
+    if: github.event_name == 'push'
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macos-13, macos-14]
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 60
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: ./.github/actions/unit-test
+        with:
+          beat: $BEAT
```

`$BEAT` will work if using `run: $BEAT` but in the context of the step, then it requires `${{ env.BEAT }}`.

I know it's a bit of cumbersome

```suggestion
          beat: ${{ env.BEAT }}
```